### DISCUSSION TITLE
多设备浏览记录不同步

### DISCUSSION THREAD

**Discussion by shoebillback** [xiaoyaocz/flutter_dmzj#129] [General]
当云端浏览记录发生变化是, 如果其他的设备已经有个本地浏览记录, 就不会同步
![IMG_9766](https://user-images.githubusercontent.com/101227862/226217149-2f6a9dec-b599-4fc7-9505-d9a8fd9a27a3.PNG)
![IMG_9765](https://user-images.githubusercontent.com/101227862/226217152-a85cc7cb-94cf-48e9-8143-6c055217bf7f.PNG)

我目前测试的设备都有这个问题, 但也不是100%触发
ios转windows
windows转ios
Android转ios

**Comment by shoebillback** [comment]
这种冲突发生时优先使用本地记录

**Comment by xiaoyaocz** [comment]
#130
